Makefile

```diff
@@ -281,6 +281,7 @@ bundle: prepare operator-sdk ## Generate bundle manifests and metadata, then val
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
 	$(OPERATOR_SDK) bundle validate ./$(BUNDLE_DIR)
 	mv bundle.Dockerfile Dockerfiles/
+	rm bundle/manifests/opendatahub-operator-webhook-service_v1_service.yaml
```

Comment:
Should we do `rm -f` to avoid failing if the file does not exist or it is ok to expect it always generated?

Reply:
let me add the force just in case